activejob/lib/active_job/test_helper.rb

```diff
@@ -75,7 +75,7 @@ def queue_adapter_for_test
     #     assert_enqueued_jobs 2
     #   end
     #
-    # If a block is passed, that block will cause the specified number of
+    # If a block is passed, asserts that the block will cause the specified number of
```

Comment:
Can you look at below/above sentences to cleanup as well, they seem to be presented in a different way.

Reply:
Agreed. Please update this to be consistent with the other sentences. As is it reads a bit weird. Also squash your commits via rebase to be only one commit.